INBOUND.md

```diff
@@ -51,8 +51,8 @@ are additional parameters consisting of name/value pairs.
 
 ### Data Items for FHIR Message Types
 
-REQUIRED - NHAIS will always provide this value for the transaction and the adaptor will always translated it FHIR
-OPTIONAL - NHAIS may optionally provide this value for the transaction and the adaptor will translate it to FHIR if provided
+REQUIRED - PCRM/NHAIS will always provide this value for the transaction and the adaptor will always translate it FHIR
+OPTIONAL - PCRM/NHAIS may optionally provide this value for the transaction and the adaptor will translate it to FHIR if provided
```

Comment:
In some places we're keeping the NHAIS name, and in some places we're not. What was the thinking?

Reply:
If you visit here: https://digital.nhs.uk/services/primary-care-registration-management
And scroll down to "Address", you see: "GP systems can send patient address updates to PCRM via [NHAIS GP Links](https://digital.nhs.uk/developer/api-catalogue/nhais-gp-links)."
So it seems that PCRM are still making some reference to an NHAIS GP Links service, so haven't quite fully committed to a name change. Seems like there is still an NHAIS service within the umbrella of PCRM.